scripts/createPage/createPage.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable */
+const path = require('path');
+const fs = require('fs');
+const { buildJS } = require('./builders');
+const {
+  containsWhiteSpace,
+  containsDashCharacter,
+  capitalizeFirstLetter,
+} = require('../../common/utils/node-utils');
+
+const pagePath = 'pages/';
+const fileNamePrefix = '.js';
+
+const findRoot = () => {
+  let thisPath = path.resolve(__dirname);
+  while (!fs.existsSync(path.join(thisPath, 'package.json'))) {
+    thisPath = path.join(thisPath, '..');
+  }
+
+  return thisPath;
+};
+
+const doesPageExist = (pageName, root) => {
+  const newPath = path.join(root, pagePath, pageName);
+  const filePrefix = '.js';
+
+  if (fs.existsSync(newPath + filePrefix)) {
+    console.log(`Page "${pageName}" Already Exists`);
+    return true;
+  }
+  return false;
+};
+
+const createPageTitle = pageName => {
```

Comment:
Just wanted to show a different implementation of this:
```
const createPageTitle = pageName => {
  // Example 1: `some_page_name` becomes `Some Page Name`
  // Example 2: `page` becomes 'Page'
  return pageName.split('_').map(word => capitalizeFirstLetter(word)).join(' ');
}
```

I wasn't sure what the conditional check on line 37 was for.

Reply:
The conditional check was for if a user named their page with an underscore in the front, e.g. _foo. When splitting a string by the underscore, it will eliminate it, but when rejoining the string you will get a space in the character in front of foo, e.g. pageTitle = ' foo'.

Reply:
I will reimplement this to match your version. (A lot cleaner)

Reply:
The conditional check was for this scenario: 
`// Example: "_some_page" becomes " Some Page"`

There will be an extra space before "Some"

Reply:
I originally had the  `createPageTitle` function like the one above. Then I thought about this possible edge case: what if a user decides to name their file name ``_some_page_name``. This will evaluate to ` `Some Page Name`` with an extra space before Some.

I am perfectly fine with doing the one above! It looks a lot cleaner and I think that edge case wouldn't affect the pageTitle in the jsx file.

Reply:
Sweet! Completely forgot about `.trim()`. Will slap it on with comments explaining the scenario I explained the examples you've mentioned.